db/seeders/20210302050835-demo-groups.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkInsert('Peoples', [{
```

Comment:
Suggestion (blocking): copy-pasta?
```suggestion
    await queryInterface.bulkInsert('Group', [{
```

Reply:
🤦🏻 I screwed up the commit. Fixed it and checked in the code I meant to